src/DssSpell.sol

```diff
@@ -90,7 +90,7 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE      = ;
 
-    uint256 internal constant ZERO_PT_SEVENTY_FIVE_PCT_RATE  = 1000000000236936036262880196;
+    uint256 internal constant ZERO_PT_SEVEN_FIVE_PCT_RATE  = 1000000000236936036262880196;
     uint256 internal constant ONE_PCT_RATE                   = 1000000000315522921573372069;
     uint256 internal constant ONE_PT_SEVENTY_FIVE_PCT_RATE   = 1000000000550121712943459312;
     uint256 internal constant THREE_PT_TWENTY_FIVE_PCT_RATE  = 1000000001014175731521720677;
```

Comment:
And this. Don't forget to change references in this *before you push* (or just run tests locally and it won't compile) so we don't have a bunch of failing CI :)